### PR TITLE
Mobilenetv2

### PR DESCRIPTION
**Pull request by zyzzu**

### CODE REVIEW

**Comment by ZwwWayne** [comment]
Hi @zyzzu ，
Thanks for your kind contribution. Please sign CLA first.

File: configs/_base_/datasets/voc0712.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # dataset settings
 dataset_type = 'VOCDataset'
-data_root = 'data/VOCdevkit/'
+data_root = '/home/SharedDatasets/VOC/'
```
Comment (ZwwWayne) [inline configs/_base_/datasets/voc0712.py:3]: Please do not modify the path of data. You can use softlink on your own machine.

**Comment by ZwwWayne** [comment]
Hi @zyzzu ,
The current implementations of Mobilenet and Shufflenet are a little bit different from their implementation in [MMSegmentation](https://github.com/open-mmlab/mmsegmentation) and [MMClassification](https://github.com/open-mmlab/mmclassification). The models of Shufflenet and Mobilenet pre-trained by MMClassification are released and can be directly used in MMDetection. So would you like to update the implementation to be consistent with MMSegmentation and MMClassification? Otherwise, this PR may not be able to be merged.

**Comment by OpenMMLab-Coodinator** [comment]
Hi @zyzzu  ！We are grateful for your efforts in helping improve this open-source project during your personal time.
Welcome to join OpenMMLab Special Interest Group (SIG) , where you can share your experiences, ideas, and build connections with like-minded peers.

To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have a WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request closed without merging**